File: Data/Models/Drivers/Driver.cs
using Infrastructure.Data.Models.Common;

namespace Infrastructure.Data.Models.Drivers
{
    class Driver
    {
        public int DriverId { get; private set; }
        public string Name { get; set; }
        public bool DaddysCash { get; set; }
        public int Pace { get; set; }
        public int Experience { get; set; }
        public int Racecraft { get; set; }
        public int Awareness { get; set; }

        public Nationality Nationality { get; set; }
        public DriverRating DriverRating { get; set; }
    }
}

File: F1Encyclopedia/ClientApp/src/app/Drivers/models/driver.ts
import { Country, Deserialisable } from "src/app/Common/models";

export class Driver implements Deserialisable {
    id: number;
    firstName: number;
    lastName: number;
    dob: Date;
    wikiUrl: string;
    country: Country;
    number: number;
    daddysCash: boolean;
    pace: number;
    experience: number;
    racecraft: number;
    awareness: number;

    deserialise(input: any): this {
        return Object.assign(this, input);
    }

    getFullName(): string {
        return `${this.firstName} ${this.lastName}`;
    }
}